Lib/test/datetimetester.py

```diff
@@ -390,6 +390,14 @@ def test_deepcopy(self):
         tz_copy = copy.deepcopy(tz)
         self.assertIs(tz_copy, tz)
 
+    def test_timezone_offset_close_twenty_four(self):
```

Comment:
Does that code actually test the python implementation? To do my local experiments, I had to comment out the lines that import the c module.

Reply:
Yeah, `datetimetester.py` is in named weirdly because the tests are actually *run* by `test_datetime.py`, which will run these tests using both the pure python `datetime` module and the C `datetime` module.